### PR TITLE
Update niv dependencies

### PR DESCRIPTION
**Pull request by Cmdv**
Running update of the niv dependencies, which should trigger new cachix.

Things still compiling on my end but will report back once done.

### CODE REVIEW

**Comment by junjihashimoto** [comment]
Thx!

**Comment by junjihashimoto** [comment]
It may be killed by the OS for not enough memory.

**Comment by junjihashimoto** [comment]
CI uses two processes. https://github.com/hasktorch/hasktorch/blob/master/.github/workflows/nix-linux.yml#L52

It may be better to use one.

**Comment by Cmdv** [comment]
ah yeah cool I'll change that now see if that's any better. Locally think it's rebuilding GHC 🤦‍♀️

**Comment by Cmdv** [comment]
damn seems to still be failing 😞

File: nix/sources.json [hunk 1]
```diff
@@ -101,10 +101,10 @@
         "homepage": "https://pytorch.org/",
         "owner": "stites",
         "repo": "pytorch-world",
-        "rev": "e40fc2133b169b72eb8d772f98b1484dc1d3903f",
+        "rev": "e08419a2f2b9ca177903c9595858b0dc386fc4dd",
```
Comment (tscholak) [inline nix/sources.json:104]: @Cmdv what commit is this? are you sure this is a newer version?

Comment (junjihashimoto) [reply nix/sources.json:104]: There is no need to change, but both are almost the same.

Comment (Cmdv) [reply nix/sources.json:104]: I literally did `niv update` and let niv change all the revs

Comment (tscholak) [reply nix/sources.json:104]: ok cool, thanks :)

**Review by tscholak — APPROVED** [review]
LGTM, let's see if the CI agrees

**Comment by tscholak** [comment]
closing since #510 has been merged

**Pull request merged**